modules/javafx.base/src/main/java/javafx/util/Subscription.java

```diff
@@ -29,8 +29,17 @@
 import java.util.Objects;
 
 /**
- * A subscription encapsulates how to cancel it without having
- * to keep track of how it was created.
+ * A {@code Subscription} represents the encapsulation of the result of an action
+ * that can be cancelled. Subscriptions can be obtained, for example, as a result
```

Comment:
The first sentence is a bit cryptic to me. I'm looking at `Runnable` that is basically a less fancy version of `Subscription` (in a way, `Subscription` could extend `Runnable`). Maybe something like
"Represents a cancel/cleanup operation for an action that can be cancelled/allocates resources".
You can also choose either.

I would also add a general example after the first paragraph, maybe like this:

```java
class Person {

    public Subscription subscribeToNewspaper(Newspaper paper) {
        register(paper);
        return () -> unregister(paper);
    }
```

Reply:
Thanks, that's a good suggestion.  I also thought the wording was a bit too hard to follow, but wanted to see what you thought.  I updated it to use your wording and added an example.